library-model/library-model-generator-cli/build.gradle

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2017. Uber Technologies
```

Comment:
Not that I care about this anymore, but I suspect Uber would want these copyright headers to be updated for the right years 😉  (e.g. `2024` here and `20[...]-2024` for the files that were altered, I think)